src/storage/store.js

```diff
@@ -18,7 +18,8 @@ export const SCHEMA = {
       additionalProperties: false,
       properties: {
         displayName: { type: "string", pattern: "^[A-Za-z0-9-]{3,32}$" },
-        avatarId: { type: "string" }
+        avatarId: { type: "string" },
+        personalAvatarId: { type: "string" }
```

Comment:
I'd consider maybe breaking this up to be a bit more future proof since these store data structures are schema checked. Perhaps an `avatars` array with right now an expected single value of `{ avatarId: XXX, type: 'personal' }`

Reply:
hmm I was assuming this would be completely throwaway and we would just be fetching this from the db. Since it represents something owned by an account I don't think its ever going to exist just in localStorage

Reply:
right now the schema validator complains if you have an unrecognized key, so once it's here, we have to keep it forever (or loosen the schema validator up somehow)

Reply:
The way our schema validation works right now seems too strict. The fact that the application will simply not start if there is a superfluous key seems wrong to me.

Reply:
i might be misremembering though